src/main/java/bio/terra/service/resourcemanagement/ResourceService.java

```diff
@@ -161,7 +161,8 @@ public GoogleBucketResource getOrCreateBucketForFile(
         flightId,
         null,
         getReaderGroups,
-        dedicatedServiceAccount);
+        dedicatedServiceAccount,
+        true);
```

Comment:
shouldn't this be based on the dataset's option?

Reply:
(I think this is covered with the other thread)